### PR TITLE
EVG-13314 Wire up clear all subscription button

### PR DESCRIPTION
**Pull request by khelif96**

### CODE REVIEW

**Review by SupaJoon — CHANGES_REQUESTED** [review]
The Clear all subscriptions button should actually remove the subscriptions created by clicking on "notify me" on the version and task pages. I created a PR for the corresponding mutation here: https://github.com/evergreen-ci/evergreen/pull/4137

**Comment by khelif96** [comment]
> The Clear all subscriptions button should actually remove the subscriptions created by clicking on "notify me" on the version and task pages. I created a PR for the corresponding mutation here: [evergreen-ci/evergreen#4137](https://github.com/evergreen-ci/evergreen/pull/4137)

Thanks!

File: src/pages/preferences/preferencesTabs/notificationTab/ClearSubscriptionsCard.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      );
+    },
+  });
+
+  return (
+    <>
+      <PreferencesCard>
+        <ContentWrapper>
+          <Body>
+            To clear all subscriptions you have made on individual Task pages.
```
Comment (SupaJoon) [inline src/pages/preferences/preferencesTabs/notificationTab/ClearSubscriptionsCard.tsx:44]: can we change this copy to "To clear all subscriptions you have made on individual Version and Task pages."?
relevant thread: https://mongodb.slack.com/archives/G013M6ULWS1/p1604515687002100

File: src/pages/preferences/preferencesTabs/notificationTab/ClearSubscriptionsCard.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  const dispatchBanner = useBannerDispatchContext();
+
+  const [clearMySubscriptions, { loading }] = useMutation<
+    ClearMySubscriptionsMutation,
+    ClearMySubscriptionsMutationVariables
+  >(CLEAR_MY_SUBSCRIPTIONS, {
+    onCompleted: (result) => {
+      setShowModal(false);
+      dispatchBanner.successBanner(
+        `Successfully cleared ${result.clearMySubscriptions} subscriptions!`
```
Comment (SupaJoon) [inline src/pages/preferences/preferencesTabs/notificationTab/ClearSubscriptionsCard.tsx:28]: These code changes work well. Can pluralize "subscriptions" based on `result.clearMySubscriptions`

**Review by SupaJoon — APPROVED** [review]

**Pull request merged**